recipe/meta.yaml

```diff
@@ -1,49 +1,84 @@
 {% set name = "maturin" %}
-{% set version = "0.11.5" %}
+{% set version = "0.13.7" %}
 
 package:
-  name: {{ name }}
+  name: {{ name }}-suite
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 07074778b063a439fdfd5501bd1d1823a216ec5b657d3ecde78fd7f2c4782422
+  sha256: c0a77aa0c57f945649ca711c806203a1b6888ad49c2b8b85196ffdcf0421db77
 
 build:
-  number: 1
-  skip: true  # [py27]
-  missing_dso_whitelist:   # [osx]
-    - /usr/lib/libresolv.9.dylib  # [osx]
+  number: 0
+  # ring (dependency loaded via cargo) is not supported on s390x yet
+  # https://github.com/briansmith/ring/issues/1511
+  skip: True  # [py<37 or s390x]
 
-requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - {{ compiler('c') }}              # [unix]
-    - {{ compiler('m2w64_c') }}        # [win]
-    # seems to work with defaults from conda-forge-pinning (at the moment)
-    - rust  # [not osx]
-    # investigating...
-    - rust >=1.54  # [osx]
-  host:
-    - pip
-    - python
-    - toml
-  run:
-    - python
-    - toml >=0.10.2,<0.11
 
-test:
-  commands:
-    - maturin --help
+outputs:
+  - name: {{ name }}
+    script: bld-win.bat                  # [win]
+    script: build-unix.sh                # [not win]
+    missing_dso_whitelist:   # [osx]
+      - /usr/lib/libresolv.9.dylib  # [osx]
+    requirements:
+      host:
+        - pip
+        - python
+        - setuptools
+        - setuptools-rust >=1.4.0
+        - wheel >=0.36.2
+        - tomli >=1.1.0
+      run:
+        - python
+        - tomli >=1.1.0
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('rust') }}     # [not win]
+        - rust_win-64                # [win]
+    test:
+      requires:
```

Comment:
Missing `maturin` in `test/imports`? See https://github.com/PyO3/maturin/blob/4c32d3c1cc69955e0516509125ad68139284f6be/setup.py#L76